fiftyone/core/annotation/attributes.py

```
@@ -0,0 +1,216 @@
+        )
+    """
+
+    operator: Union[WhenOperator, str]
+    field: str
+    value: Any
+    then: Optional[dict] = None
+
+    def __post_init__(self) -> None:
+        self.operator = WhenOperator(self.operator)
+        if not self.field:
+            raise ValueError("When.field is required")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)